netci.groovy

```diff
@@ -2232,8 +2267,19 @@ Constants.allScenarios.each { scenario ->
                     def osGroup = getOSGroup(os)
                     def jobName = getJobName(configuration, architecture, os, scenario, false) + "_tst"
 
+                    def isArmStressConfiguration = false
+                    if (lowerConfiguration != 'default' && (architecture in validWindowsNTCrossArches)) {
+                        isArmStressConfiguration = true
+                    }
+
+                    def coreclrBuildLookupConfiguration = 'default'
+
+                    if (isArmStressConfiguration && lowerConfiguration != 'default') {
```

Comment:
I don't think you need the `isArmStressConfiguration` variable at all, just change this expression to:
```
if ((lowerConfiguration != 'default') && (architecture in validWindowsNTCrossArches))
```
(calling it a "stress" configuration is a little confusing anyway).